terraform/modules/paas/main.tf

```diff
@@ -0,0 +1,44 @@
+resource cloudfoundry_service_instance postgres_instance {
+  name         = local.postgres_service_name
+  space        = data.cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.postgres.service_plans["${var.postgres_service_plan}"]
+  json_params  = "{\"enable_extensions\": [\"pgcrypto\", \"fuzzystrmatch\", \"plpgsql\"]}"
+}
+
+resource cloudfoundry_service_instance redis_instance {
+  name         = local.redis_service_name
+  space        = data.cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.redis.service_plans["${var.redis_service_plan}"]
+}
+
+resource cloudfoundry_app web_app {
+  name                       = local.web_app_name
+  command                    = local.web_app_start_command
+  docker_image               = var.app_docker_image
+  health_check_type          = "http"
+  #health_check_http_endpoint = "" #TODO Change this to the correct healthcheck endpoint
+  instances                  = var.web_app_instances
+  memory                     = var.web_app_memory
+  ports    = [local.web_app_runs_on_port,]
+  space    = data.cloudfoundry_space.space.id
+  stopped  = var.app_stopped
+  strategy = var.web_app_deployment_strategy
+  timeout  = var.app_start_timeout
+  service_binding {
+    service_instance = cloudfoundry_service_instance.postgres_instance.id
+  }
+  service_binding {
+    service_instance = cloudfoundry_service_instance.redis_instance.id
+  }
+  environment = local.app_environment_variables
+}
+
+resource cloudfoundry_route web_app_route {
+  domain   = data.cloudfoundry_domain.cloudapps_digital.id
+  space    = data.cloudfoundry_space.space.id
+  hostname = local.web_app_name
+  target {
+    app  = cloudfoundry_app.web_app.id
+    port = local.web_app_runs_on_port
```

Comment:
Can we use the same model as TV: https://github.com/DFE-Digital/teacher-vacancy-service/blob/master/terraform/app/modules/paas/main.tf#L47
ie the app binds to the route instead of the route binds to the app